solana/src/lib.rs

```diff
@@ -0,0 +1,23 @@
+
+use cvt_core::CVT_assume;
+
+// We redefine this macro to avoid including error code conversion/formatting
```

Comment:
The comment says that we re-define something, but the code does not re-define, it just defines.

This probably indicates that we need to shadow some library (maybe solana-program) and in that shadow implementation we replace the definition of a macro by ours. 

This might need more discussion to understand what is the best solution.